src/Cell.tsx

```diff
@@ -102,7 +103,7 @@ export class Cell<Data: Types.CellBase, Value> extends PureComponent<
     const { row, column, getValue, formulaParser } = this.props;
     let { DataViewer, data } = this.props;
     if (data && data.DataViewer) {
-      ({ DataViewer, ...data } = data);
+      DataViewer = data.DataViewer;
```

Comment:
```suggestion
      { DataViewer } = data;
```
Destructuring is still cleaner